nsx/config.go

```diff
@@ -0,0 +1,82 @@
+package nsx
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/IBM-tfproviders/govnsx"
+)
+
+type Config struct {
+	User          string
+	Password      string
+	NsxManagerUri string
+	InsecureFlag  bool
+	Debug         bool
+	DebugPath     string
+	DebugPathRun  string
+}
+
+//
+// Client() returns a new client for accessing NSX manager
+//
+func (c *Config) Client() (*govnsx.Client, error) {
+
+	err := c.EnableDebug()
```

Comment:
even before we create the client object create, we need to set the debug related facility to cover the client object creation code too